components/local-app/cmd/config-set.go

```diff
@@ -55,10 +86,18 @@ Example:
 var configSetOpts struct {
 	Autoupdate bool
 	Telemetry  bool
+
+	Host           string
+	OrganizationID string
+	Token          string
```

Comment:
You can set these three, but can't `get` them. I think it would be nice to update the get command for them as well.